<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutlineForm</class>
 <widget class="QWidget" name="OutlineForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>118</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ptChoice" name="OverlayMode"/>
     </item>
     <item>
      <widget class="ptCheck" name="ImageOnTop">
       <property name="text">
        <string notr="true">CheckBox</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ptChoice" name="GradientMode"/>
   </item>
   <item>
    <widget class="ptInput" name="ColorWeight"/>
   </item>
   <item>
    <widget class="ptInput" name="BlurRadius"/>
   </item>
   <item>
    <widget class="ptCurveWindow" name="Curve" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ptCheck</class>
   <extends>QCheckBox</extends>
   <header>ptCheck.h</header>
  </customwidget>
  <customwidget>
   <class>ptInput</class>
   <extends>QProgressBar</extends>
   <header>ptInput.h</header>
  </customwidget>
  <customwidget>
   <class>ptChoice</class>
   <extends>QComboBox</extends>
   <header location="global">ptChoice.h</header>
  </customwidget>
  <customwidget>
   <class>ptCurveWindow</class>
   <extends>QWidget</extends>
   <header location="global">ptCurveWindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
